python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+        'options'          : GangaFileItem ( defvalue=[], sequence=1, hidden = 1,doc='List of files which contain the options to pass to gaudirun.py'),
+        'extraArgs'        : SimpleItem(defvalue=[], typelist=[list], sequence=1, hidden=1, doc=' Do NOT USE IT; Extra runtime arguments which are passed to the code running on the WN'),
+        # Prepared job object
+        'is_prepared'      : SimpleItem(defvalue=None, strict_sequence=0, visitable=1, copyable=1, hidden=0, typelist=[None, ShareDir], protected=0, comparable=1,
+                                        doc='Location of shared resources. Presence of this attribute implies the application has been prepared.'),
+        'hash'             : SimpleItem(defvalue=None, typelist=[None, str], hidden=1, doc='MD5 hash of the string representation of applications preparable attributes'),
+        })
+    _category      = 'applications'
+    _name          = 'BenderBox'
+    _exportmethods = ['prepare', 'unprepare' ]
+
+    def configure(self, masterjobconfig):	
+        self.options = self.tool.inputfiles()
+        return (None,None)
+
+    # =======================================================================
+    def getWNPythonContents(self):
+        """Return the wrapper script which is used to run Bender on the WN
+        """
+        from ..RTHandlers.GaudiExecRTHandlers import GaudiExecDiracRTHandler
+        data_file = GaudiExecDiracRTHandler.data_file
+        return self.tool.getWrapper(data_file)
+
+
+# =============================================================================
+## prepare Bender application
+#  @code
+#  j = Job ( application = prepareBender ( 'v30r1' ) ) 
+#  @endcode
+#  One can specify the directory explicitely:
+#  @code
+#  j = Job ( application = prepareBender ( 'v30r1' , path = '$HOME/mydir' ) ) 
+#  @endcode
+#  or use the temporary directory 
+#  @code
+#  j = Job ( application = prepareBender ( 'v30r1' , use_tmp = True  ) ) 
+#  @endcode
+#  One can also use the configuration parameters:
+#  @code
+#  j = Job ( application = prepareBender ( 'v30r1' , module = 'the_path/the_module.py' , params = { ...} ) ) 
+#  @endcode
+def prepareBender ( version  , path = '$HOME/cmtuser', use_tmp = False , **kwargs ) :
+    """Prepare Bender application
+    >>> j = Job ( application = prepareBender ( 'v30r1' ) )
+    One can specify the directory explicitely:
+    >>> j = Job ( application = prepareBender ( 'v30r1' , path = '$HOME/mydir' ) ) 
+    or use temporary directory: 
+    >>> j = Job ( application = prepareBender ( 'v30r1' , use_tmp = True  ) ) 
+    One can also use the configuration parameters:
+    >>> j = Job ( application = prepareBender ( 'v30r1' , module = 'the_path/the_module.py' , params = { ...} ) ) 
+    """
+    from .GaudiExecUtils import prepare_cmake_app
+    if use_tmp :
+        import tempfile
+        path = tempfile.mkdtemp ( prefix = 'GANGA_BENDER_%s_' % version )
+        from Ganga.Utility.logging import getLogger
```

Comment:
it's a PEP8 thing: [PEP8 imports](https://www.python.org/dev/peps/pep-0008/#imports)